src/core/src/core_logic/PatchInstaller.py

```diff
@@ -177,14 +179,48 @@ def install_updates(self, maintenance_window, package_manager, simulate=False):
         self.last_still_needed_packages = all_packages
         self.last_still_needed_package_versions = all_package_versions
 
+        batch_patching_stopwatch = Stopwatch(self.env_layer, self.telemetry_writer, self.composite_logger)
+        batch_patching_stopwatch.start()
+
+        installed_update_count, patch_installation_successful, maintenance_window_batch_cutoff_reached = self.install_patches_in_batches(all_packages, packages, package_versions, maintenance_window, package_manager)
```

Comment:
Rename install_patches_in_batches() to install_updates_in_batches() or install_packages_in_batches()

NOTE: If you decide to use install_packages_in_batches(), please rename update to package everywhere in this function/code path

Reply:
Updated to install_packages_in_batches.

Using package instead of patch and update in the new code. The following are some of the variables in existing code with patch or update in name. I am not changing them to be consistent with existing code. 
patch_installation_successful and overall_patch_installation_successful
installed_update_count
successful_parent_update_count, failed_parent_update_count and attempted_parent_update_count